operations/csm-scheduled-tasks/cmd/server/main.go

```
@@ -0,0 +1,243 @@
+}
+
+func mustEnv(key string) string {
+	v := os.Getenv(key)
+	if v == "" {
+		slog.Error("required environment variable is not set", "key", key)
+		os.Exit(1)
+	}
+	return v
+}
+
+func envOrDefault(key, def string) string {
```

golangci-lint (2.12.2)

[error] func envOrDefault is unused

(unused)